### DISCUSSION TITLE
[Nitpick] Portal feature requests

### DISCUSSION THREAD

**Discussion by scjoiner** [TeamRizu/OutFox#522] [03 - Feature Requests]
1. I can't see my high scores when selecting a song using Portal mode since the "Portal" selection stays highlighted on the music wheel. 

2. I can't tell what game the portal song is from until the song is over

3. The portal mode dumps you into the song group after the song is completed, so if I want to play another random song, I have to close that group and go back to the portal. It would be preferable if you returned to the Portal mode after competing a song in order to select another.

**Comment by Joehuu** [comment]
1. You can see your grade + percent per diff on the bottom left side of the screen
2. Not sure what you mean about the "game the portal song is". There's a preview of the metadata showing before and when you are playing the song
3. Duplicate of https://github.com/TeamRizu/OutFox/issues/217

**Comment by scjoiner** [comment]
Hey Joseph,
To clarify #2, by game I mean which list the song belongs to, which in my case corresponds to the game the song originates from. 

Thanks
Stephen

**Comment by scjoiner** [comment]
Joseph, it looks like the high score is immediately hidden by the song preview nps graph. So if you don’t catch it right away you aren’t able to see it. 

Thanks!
